core/metrics-core-service/src/test/java/org/hawkular/metrics/core/service/metrics/ExpressionTagQueryITest.java

```diff
@@ -121,6 +121,38 @@ public void tagValueSearch() throws Exception {
     }
 
     @Test
+    public void tagValueSearchWithDot() throws Exception {
+        String tenantId = "jsonT1Tag";
+        createTagMetrics(tenantId);
+
+        ExpressionTagQueryParser test = new ExpressionTagQueryParser(dataAccess, metricsService);
+
+        List<Metric<Long>> counters = test.parse(tenantId, COUNTER, "a2.label1 =5").toList().toBlocking()
+                .lastOrDefault(null);
+        assertMetricListById(counters, "c2");
+
+        counters = test.parse(tenantId, COUNTER, "a2.label1 = '5'").toList().toBlocking()
+                .lastOrDefault(null);
+        assertMetricListById(counters, "c2");
+
+        counters = test.parse(tenantId, COUNTER, "a2.label1 = '5.6.7'").toList().toBlocking()
+                .lastOrDefault(null);
+        assertMetricListById(counters, "c3");
+
+        counters = test.parse(tenantId, COUNTER, "a2.label1 = 5.6.7").toList().toBlocking()
+                .lastOrDefault(null);
+        assertMetricListById(counters, "c3");
+
+        counters = test.parse(tenantId, COUNTER, "a2.label1").toList().toBlocking()
+                .lastOrDefault(null);
+        assertMetricListById(counters, "c2", "c3");
+
+        counters = test.parse(tenantId, COUNTER, "a2.label1 = '5.*'").toList().toBlocking()
```

Comment:
The intention here was to match anything that starts with `5`, including `5` as value.